### PR TITLE
GT.0.2.0

### PR DESCRIPTION
**Pull request by Kakadu**

### CODE REVIEW

File: packages/GT/GT.0.2.0/opam [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+maintainer: "kakadu.hafanana@gmail.com"
+authors: ["https://github.com/dboulytchev" "https://github.com/Kakadu"]
+homepage: "https://github.com/JetBrains-Research/GT"
+bug-reports: "https://github.com/JetBrains-Research/GT/issues"
+depends: [
+  "ocaml"
+  "ocamlfind"  {build}
+  "ocamlbuild" {build}
+  "ocamlgraph" {build}
```
Comment (kit-ty-kate) [inline packages/GT/GT.0.2.0/opam]:
```suggestion
  "ocamlgraph"
```
According to your META file, ocamlgraph is required as a library, it is not a build dependency.

File: packages/GT/GT.0.2.0/opam [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  "ppxlib"     {>= "0.9.0" }
+  "camlp5"
+  "ppx_import" {with-test}
+]
+build: [
+  [make]
+  [make "test"] {with-test}
+]
+install: [make "PREFIX=%{prefix}%" "install"]
+remove: [make "PREFIX=%{prefix}%" "uninstall"]
```
Comment (kit-ty-kate) [inline packages/GT/GT.0.2.0/opam]:
```suggestion
```
This field is unnecessary since opam 2.0

**Comment by kit-ty-kate** [comment]
Build fail on 4.04 and 4.05 with:
```
# File "camlp5/Camlp5Helpers.ml", line 427, characters 6-15:
# Error: Unbound value List.init
# Command exited with code 2.
# make[1]: *** [Makefile:33: compile] Error 10
# make[1]: Leaving directory '/home/opam/.opam/4.04/.opam-switch/build/GT.0.2.0'
# make: *** [Makefile:28: all] Error 2
```
On 4.06 with:
```
# + ocamlfind ocamlc -c -g -bin-annot -short-paths -I src -package compiler-libs.common -warn-error +5 -w -13 -I regression -o regression/test810cool.cmo regression/test810cool.ml
# findlib: [WARNING] Interface topdirs.cmi occurs in several directories: /home/opam/.opam/4.06/lib/ocaml, /home/opam/.opam/4.06/lib/ocaml/compiler-libs
# File "regression/test810cool.ml", line 5, characters 4-176:
# Error: This variant or record definition does not match that of type
#          Types.type_expr
#        Fields number 3 have different names, id and scope.
# Command exited with code 2.
# make[1]: *** [Makefile:33: compile] Error 10
# make[1]: Leaving directory '/home/opam/.opam/4.06/.opam-switch/build/GT.0.2.0'
# make: *** [Makefile:137: tests] Error 2
```

File: packages/GT/GT.0.2.0/opam (hunk 2 above)
Comment (Kakadu) [reply packages/GT/GT.0.2.0/opam]: @kit-ty-kate Do you think that we should make it reportable by `opam lint`?

**Comment by kit-ty-kate** [comment]
Thanks

**Pull request merged**